lib/src/config/route/app_routes.dart

```diff
@@ -0,0 +1,15 @@
+import 'package:flutter/material.dart';
+
+import '../../data/models/movie_model.dart';
+import '../../presentation/pages/movie_details.dart';
+
+abstract class AppRoutes {
+  static Map<String, WidgetBuilder> get routes {
```

Comment:
I'd put the "HomePage" route here as well.

Reply:
Done